WeakAuras/Prototypes.lua

```diff
@@ -4569,12 +4613,16 @@ WeakAuras.event_prototypes = {
   ["Threat Situation"] = {
     type = "status",
     events = function(trigger)
-      local result = {
-        "UNIT_THREAT_SITUATION_UPDATE",
-      };
-      AddUnitChangeEvents(trigger.threatUnit, result);
+      local result = {};
+      AddUnitEventForEvents(trigger.threatUnit, result, "UNIT_THREAT_SITUATION_UPDATE")
```

Comment:
I'm not 100% confident on that one

Reply:
Can you test it?

Reply:
It might be that you really want AddUnitEventForEvents("player", result, "UNIT_THREAT_SITUATION_UPDATE") here

Reply:
You are right and then
```Lua
    events = {},
    unit_events = {
      ["player"] = {"UNIT_THREAT_SITUATION_UPDATE"}
    },
```
is correct